libcodechecker/log/option_parser.py

```diff
@@ -134,7 +134,11 @@
     '-sectorder': 3,
     '--param': 1,
     '-u': 1,
-    '--serialize-diagnostics': 1
+    '--serialize-diagnostics': 1,
+    # If Clang gives warning on which GCC doesn't then during checking it is
+    # considered an error and fails the analysis if the original build contains
+    # -Werror flag.
```

Comment:
This is not a well written English sentence.
I suggest rewriting the comment to:
"Clang gives different warnings than GCC. Thus if this flag is kept, the analysis with Clang can fail even if the compilation passes with GCC. "